content/en/tracing/trace_collection/compatibility/go.md

```diff
@@ -17,11 +17,11 @@ further_reading:
 
 The Go Datadog Trace library is open source. View the [GitHub repository][1] for more information.
 
-The Go Datadog Trace Library has a [version support policy][2] defined for Go versions. The two latest releases of Go are fully supported, while the third newest release is considered in [maintenance][3]. Older versions may function, but no support is provided by default. For special requests, [contact support][4]. 
+The Go Datadog Trace Library has a [version support policy][2] defined for Go versions. The two latest releases of Go are fully supported, while older versions may function, but no support is provided by default. For special requests, [contact support][4]. 
```

Comment:
@nsrip-dd I agree we can drop this, but I feel too that it's good to highlight that older versions may function. That can be because of the non-advisory directive from 1.21, or because the costumer's Go toolchain doesn't have the expected stdlib packages or is unable to compile language additions like generics. In both cases the statement "older versions may function" seems to apply.